packages/react/index.js

```diff
@@ -50,4 +50,7 @@ module.exports = (neutrino, opts = {}) => {
   });
 
   neutrino.config.resolve.alias.set('react-native', 'react-native-web');
+
+  neutrino.config.resolve.extensions.prepend('.web.js');
```

Comment:
Could you add a comment linking to the react-native-web docs, and also a `Fixes #1056` to the commit message/PR description? :-)

Reply:
Done.

Reply:
Hello, I am trying hard to distribute react-native packages with extensions support from `node_modules`,  `.web.js`, `.android.js`, `.js` , I don't get how to tell react-native application to import the correct file when `node_modules`. I am not able to tell the webapp to import `.web.js` first,  too  when import happen from `node_modules`, I initialized my project with expo SDk36, do you know how I can solve my issue? Thanks again for reading.